<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimDSP</class>
 <widget class="QMainWindow" name="SimDSP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimDSP</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/images/simdsp_icon.png</normaloff>../resources/images/simdsp_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="mainLayout" stretch="4,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::East</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="codeTab">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>Code</string>
         </attribute>
         <layout class="QHBoxLayout" name="codeLayout" stretch="0,5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="SDProjectexplorer" name="widgetProject" native="true"/>
          </item>
          <item>
           <widget class="SDEditortab" name="widgetEditor" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="runTab">
         <attribute name="title">
          <string>Run</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="runLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="appOutput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionNewFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionBuildProject"/>
    <addaction name="actionCleanProject"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionIncreaseFontSize"/>
    <addaction name="actionDecreaseFontSize"/>
    <addaction name="actionResetFontSize"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionSaveFile"/>
   <addaction name="separator"/>
   <addaction name="actionBuildProject"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../simdspresources.qrc">
     <normaloff>:/resources/images/icons/load_mat.png</normaloff>:/resources/images/icons/load_mat.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load Mat File</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="../simdspresources.qrc">
     <normaloff>:/resources/images/icons/new_file.png</normaloff>:/resources/images/icons/new_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New File</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../simdspresources.qrc">
     <normaloff>:/resources/images/icons/play.png</normaloff>:/resources/images/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../simdspresources.qrc">
     <normaloff>:/resources/images/icons/stop.png</normaloff>:/resources/images/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionBuildProject">
   <property name="icon">
    <iconset resource="../simdspresources.qrc">
     <normaloff>:/resources/images/icons/builder.png</normaloff>:/resources/images/icons/builder.png</iconset>
   </property>
   <property name="text">
    <string>Build Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New &amp;Project</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="../simdspresources.qrc">
     <normaloff>:/resources/images/icons/save.png</normaloff>:/resources/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../simdspresources.qrc">
     <normaloff>:/resources/images/icons/save_mat.png</normaloff>:/resources/images/icons/save_mat.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Mat File</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="toolTip">
    <string>Help Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionIncreaseFontSize">
   <property name="text">
    <string>Increase Font Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionDecreaseFontSize">
   <property name="text">
    <string>Decrease Font Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionResetFontSize">
   <property name="text">
    <string>Reset Font Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionCleanProject">
   <property name="text">
    <string>Clean Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SDEditortab</class>
   <extends>QWidget</extends>
   <header>../src/editor/sdeditortab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SDProjectexplorer</class>
   <extends>QWidget</extends>
   <header>../src/project/sdprojectexplorer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../simdspresources.qrc"/>
 </resources>
 <connections/>
</ui>
